docs/index.rst

```diff
@@ -0,0 +1,38 @@
+******************************
+One Code Python Client Library
+******************************
+
+Introduction
+============
+
+This package provides 3 major pieces of functionality: (1) a core Python client
+library; (2) a simple CLI for interacting with the `One Codex platform
+<https://onecodex.com>`_ that uses that core library; and (3) optional
+extensions to the client library, which offers many features aimed at advanced
+users and provides functionality for use in interactive notebook environments
+(e.g., Jupyter notebooks).
+
+For installation instructions and instructions on using the command-line
+client, see the :doc:`readme_link`. You can find the source code on `GitHub
+<https://github.com/onecodex/onecodex>`_.
+
+Quickstart
+==========
+
+For documentation and examples of common uses of the client library including
+interactive data analysis and plotting within Jupyter notebooks see
+:doc:`Notebooks Demo <notebook_examples/notebooks_demo>`.
+
+API Documentation
+=================
+
+For detailed documentation of the Python library, see :ref:`modindex`. For
+documentation related to results retrieval, data analysis and plotting
+functionality, see :mod:`onecodex.models.collection`.
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+   readme_link
```

Comment:
It's a little confusing to have the readme (with install instructions, basic usage, etc.) appearing in this "API Documentation" list. Maybe the readme can have its own section, e.g. "Getting Started" ?